src/openzaak/components/documenten/migrations/0007_install_uuid_extension.py

```diff
@@ -0,0 +1,11 @@
+# Generated by Django 2.2.4 on 2019-08-22 14:38
+
+from django.db import migrations
+from django.contrib.postgres.operations import CreateExtension
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("documenten", "0006_auto_20190822_1327")]
+
+    operations = [CreateExtension("uuid-ossp")]
```

Comment:
this is no longer needed - entire migration can be deleted :fire:

Reply:
Thanks! I forgot to clean it